.github/workflows/ci.yml

```
@@ -146,8 +146,9 @@ jobs:
         sudo apt-get update -y
         sudo apt-get install -y libhwloc-dev
     - name: Before script
+      # With [^\s]* we also remove possible -flto= stuff
       run: |
-        sed -i 's/-O3 -flto/-Og -fsanitize=address,undefined/' makemake.sh
+        sed -i 's/-O3 -flto[^\s]*/-Og -fsanitize=address,undefined/' makemake.sh
```

Backslash character classes like `\s` are not portable. You would need to use POSIX character classes instead:
```suggestion
        sed -i 's/-O3 -flto[^[:space:]]*/-Og -fsanitize=address,undefined/' makemake.sh
```